packages/peregrine/lib/talons/CategoryTree/useCategoryBranch.js

```diff
@@ -0,0 +1,25 @@
+import { useCallback } from 'react';
+
+/**
+ * Returns props necessary to render a CategoryBranch component.
```

Comment:
I love these comments. I'm going to go through and update my existing PRs with doc comments as well.